src/coreclr/src/tools/aot/ILCompiler.Reflection.ReadyToRun/DebugInfo.cs

```diff
@@ -68,7 +68,6 @@ public static string GetPlatformSpecificRegister(Machine machine, int regnum)
                 case Machine.Amd64:
                     return ((Amd64.Registers)regnum).ToString();
                 case Machine.Arm:
-                case Machine.ArmThumb2:
```

Comment:
It appears that the script is overwriting some expected changes like this one. We need to make sure we only add the expected changes.